spec/support/initializers/capybara.rb

```diff
@@ -20,6 +20,12 @@
     driven_by :rack_test
   end
 
+  config.around(:each, raise_server_errors: false) do |example|
```

Comment:
Do we know why the images fail to load though? I'm only asking as we'll need them once Percy starts running again.

Reply:
I am not sure why this doesnt happen all the time but we load the image path into the HTML/javascript and **then my best guess** would be something like InstantClick then tries to follow it but ends up trying to run the path against our server instead of through a helper like cloudinary which would resolve it correctly. There is probably some host env variable we can set to ensure the pictures always get loaded but I have a lot of specs that need fixing right now and this seemed like the easiest and quickest fix. At this point I dont think Percy will be coming back anytime soon but when it does, if this fails those snapshots will report the problem and we can address it then more "correctly"